docs/Formatting.md

```diff
@@ -395,4 +395,22 @@ If it is set to `true`, the above document becomes:
     <child />
   </robot>
   ```
-***
\ No newline at end of file
+***
+### xml.format.closingBracketNewLine
+
+If set to `true`, the closing bracket (`>` or `/>`) of a tag with at least 2 attributes will be put on a new line.
+
+Requires [splitAttributes](#xml.format.splitAttributes) to be set to `true`.
```

Comment:
This link does not work. I think if you remove the `.` from it, it should work.

Reply:
It seems to work on vscode on my side, are you using it through the gh ui?

Reply:
I'm click on in from within the internal documentation